addons/account/static/src/less/account_journal_dashboard.less

```diff
@@ -45,6 +45,58 @@
     }
 }
 
+// Style for Bank Statement line graph
+.o_journal_bank{
+    .o_dashboard_graph {
+        &.o_graph_linechart {
+            > svg {
+                height: 75px;
+
+                // X axis
+                g.nv-x.nv-axis {
+                    transform: translateY(50px);
+
+                    // Remove max and min labels
+                    g.nv-axisMaxMin {
+                        display: none;
+                    }
+                    g.tick.major {
+                        // Don't know why, but leads to strange bold text
+                        // stroke: @odoo-brand-primary;
+
+                        line {
+                            display: none;
+                        }
+                    }
+                }
+
+                // !important are needed because nvd3 uses inline style on elements
+
+                // Remove strokes
+                g.nv-linesWrap {
+                    g.nv-group.nv-series-0 {
+                        fill-opacity: 0.3 !important;
+                        fill: @odoo-brand-primary !important;
+                        stroke: @odoo-brand-primary !important;
+
+                        .nv-point {
+                            visibility: hidden;
+                            r: 3;
+                            stroke: @odoo-brand-primary !important;
+                            fill: white;
+                            stroke-width: 2;
+                        }
+                        .nv-point:nth-child(n+1) {
+                            visibility: visible;
+                            fill-opacity: .95 !important;
+                            stroke-opacity: .95 !important;
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
I think this kind of changes in the `.less` is not mandatory by the specs or I miss something?